pkg/kubelet/kubelet.go

```diff
@@ -256,7 +256,7 @@ type Dependencies struct {
 	EventClient             v1core.EventsGetter
 	HeartbeatClient         clientset.Interface
 	OnHeartbeatFailure      []func()
-	KubeClient              clientset.Interface
+	KubeClient              clientset.Interface //TODO: remove
```

Comment:
this is here for backward compatibility. will be removed in the next PR when kubelet starts to use KubeTPClients instead

Reply:
kubelet changes is relatively not very large if not including the UT effort etc. it probably make a sense to include the current POC code changes in one PR and then add on another PR for UT and other testing related effort?

Reply:
the current POC left plenty of missing pieces that would not be suitable for the master branch as is. that's why this is split into pieces to control impact.